docs/reference/indices/put-component-template.asciidoc

```diff
@@ -91,23 +91,25 @@ Name of the component template to create.
 {es} includes the following built-in component templates:
 
 // tag::built-in-component-templates[]
-- `logs-mappings`
-- `logs-settings`
-- `metrics-mappings`
-- `metrics-settings`
-- `metrics-tsdb-settings`
-- `synthetics-mapping`
-- `synthetics-settings`
+- `logs@mappings`
+- `logs@settings`
+- `metrics@mappings`
+- `metrics@settings`
+- `metrics@tsdb-settings`
+- `synthetics@mapping`
+- `synthetics@settings`
 // end::built-in-component-templates[]
 
 {fleet-guide}/fleet-overview.html[{agent}] uses these templates to configure
-backing indices for its data streams. If you use {agent} and want to overwrite
-one of these templates, set the `version` for your replacement template higher
-than the current version.
-
-If you don't use {agent} and want to disable all built-in component and index
-templates, set <<stack-templates-enabled,`stack.templates.enabled`>> to `false`
-using the <<cluster-update-settings,cluster update settings API>>.
+backing indices for its data streams.
+If you want to customize these templates, don't override them as they may be reset after an update.
+Instead, look for a `...@custom` component template in the `composed_of` section of the managed index template.
+These custom component templates allow you to customize the mappings of managed index templates,
+without having to override managed index templates or component templates.
```

Comment:
These templates may not exist yet, maybe it's worth mentioning this?

Reply:
I've added some sentences. WDYT?